### PR TITLE
Fix Emotion theming in Next.JS app

### PR DESCRIPTION
**Pull request by amoore108**
Overall changes
======
Fixes an issue with working with multiple `node_modules` locally. The Next.JS app is importing Emotion components from the old app, so it appears to be getting confused about which copy of Emotion to use.

This fix adds in module resolution to the Next.JS apps Jest and `next.config.js` file to ensure that any references to `@emotion/react` should use the Next.JS compiled version, rather than the Express one. Not ideal but works.

Also adds basic styling to test theming and to check page weight increases with more CSS introduced.

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: ws-nextjs-app/pages/[service]/new_live/[id]/Posts/styles.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../../../../../src/app/utilities/pixelsToRem';
+
+export default {
+  postItem: ({ palette }: Theme) =>
+    css({
+      backgroundColor: palette.WHITE,
+      padding: `${pixelsToRem(16)}rem`,
```
Comment (andrewscfc) [inline ws-nextjs-app/pages/[service]/new_live/[id]/Posts/styles.ts:8]: Is some UX work going on here or is this just a bit of placeholder styling?

**Review by andrewscfc — APPROVED** [review]

File: ws-nextjs-app/pages/[service]/new_live/[id]/Posts/styles.ts (hunk 1 above)
Comment (amoore108) [reply ws-nextjs-app/pages/[service]/new_live/[id]/Posts/styles.ts:8]: Yea all very much placeholder. Just wanted to add something to see any differences in theme provided CSS adding to the page weight.

File: ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+/** @jsx jsx */
```
Comment (karinathomasbbc) [inline ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx:1]: Does this also need the classicRuntime? (Had you already defaulted that for the NextJS app?)

File: ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx [hunk 3]
```diff
@@ -1,6 +1,7 @@
+/* * @jsxRuntime classic */
```
Comment (karinathomasbbc) [inline ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx]: I can't remember if this was always set by default, so you don't need to do it here?

**Review by karinathomasbbc — APPROVED** [review]

File: ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx (hunk 3 above)
Comment (amoore108) [reply ws-nextjs-app/pages/[service]/new_live/[id]/LivePageLayout.tsx]: Good point, think I can remove this.

File: ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx (hunk 2 above)
Comment (amoore108) [reply ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx:1]: Don't think we will need this now as mentioned below.

**Comment by amoore108** [comment]
Closing this in favour of https://github.com/bbc/simorgh/pull/10868

**Pull request closed without merging**